data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1212,12 +1212,13 @@
     "subcategory": "CSC_CHEM_CHEMICALS",
     "skill_used": "chemistry",
     "difficulty": 4,
+    "result_mult": 4,
     "time": "1 h",
     "batch_time_factors": [ 80, 4 ],
     "autolearn": true,
     "qualities": [ { "id": "BOIL", "level": 2 }, { "id": "DISTILL", "level": 1 } ],
     "tools": [ [ [ "surface_heat", 25, "LIST" ] ] ],
-    "components": [ [ [ "splinter", 40 ], [ "2x4", 12 ] ], [ [ "water", 3 ], [ "water_clean", 3 ] ] ]
+    "components": [ [ [ "splinter", 40 ], [ "2x4", 2 ] ], [ [ "water", 3 ], [ "water_clean", 3 ] ] ]
```

Comment:
```suggestion
    "components": [ [ [ "splinter", 55 ] ], [ [ "water", 50 ], [ "water_clean", 50 ] ] ]
```

Reply:
I also removed all the planks as you can't use this method with wood that hasn't been broken down into pellet sizes.  And one hour doesn't get the wood broken down and processed into methanol from planks.